core/src/main/java/nl/nn/adapterframework/errormessageformatters/FixedErrorMessage.java

```diff
@@ -58,7 +59,11 @@ public String format(String errorMessage, Throwable t, INamedObject location, Me
 			}
 		}  
 		if (StringUtils.isEmpty(stringToReturn)) {
-			stringToReturn = super.format(errorMessage, t, location, originalMessage, messageId, receivedTime);
+			try {
+				stringToReturn = super.format(errorMessage, t, location, originalMessage, messageId, receivedTime).asString();
```

Comment:
Dit kan ook messageToReturn zijn toch? Ik zie niet in waarom onderstaande `XmlUtils.transformXml(...)` niet met een message zou kunnen.